creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -62,45 +81,89 @@ class SyncHistoryAggregator {
     }
   }
 
+  /**
+   * Retrieves the current key's ttl. If the key doesn't exist or is indefinite,
```

Comment:
do you mean if key doesn't exist or is zero in this comment? i don't see any check for key is indefinite

Reply:
no, by default, all redis keys are indefinite if no ttl is assigned. a ttl that is indefinite has a value of `-1`

Reply:
oh cool, gotchu